packages/vite/src/node/config.ts

```diff
@@ -606,7 +606,7 @@ function resolveBaseUrl(
   if (isExternalUrl(base)) {
     if (!isBuild) {
       // get base from full url during dev
-      const parsed = parseUrl(base)
+      const parsed = new URL(base, 'http://vitejs.dev')
```

Comment:
Did you mean to add the second argument?

Reply:
The `base` maybe a relative-path, and the second argument is required 😄

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters

Reply:
Maybe we'll need a constant dummy URL that we pass around. It is very surprising to see the Vite homepage in the code 😬

Reply:
May use `/` better 😂